src/hooks/useFetch.ts

```diff
@@ -35,6 +35,7 @@ export const useFetch = <T>(client: AxiosInstance, config: Config): [T | undefin
 
 export const useLazyFetch = <T>(client: AxiosInstance): [<Q>(config: Config) => Promise<AxiosResponse<Q, any>>, { loading: boolean, data: T | undefined, headers: AxiosResponseHeaders | undefined }] => {
   const [data, setData] = useState<T>();
+  // setHeaders is not used
   const [headers, setHeaders] = useState<AxiosResponseHeaders>();
```

Comment:
https://github.com/opentdf/frontend/actions/runs/6882087409/job/18719884190?pr=212#step:5:20

```suggestion
  const [headers] = useState<AxiosResponseHeaders>();
```

Reply:
ugh, this does have an used cascade effect.  looking....

Reply:
ok, fixed type